tutorials/platform/consoles.rst

```diff
@@ -66,7 +66,7 @@ Following is the list of providers:
 - `Lone Wolf Technology <http://www.lonewolftechnology.com/>`_ offers
   Switch and PS4 porting and publishing of Godot games.
 - `Pineapple Works <https://pineapple.works/>`_ offers
-  Switch, Xbox One and Xbox Series X/S porting and publishing of Godot games.
+  Switch, Xbox One & Xbox Series X/S (GDK) porting and publishing of Godot games (GDScript/C#).
```

Comment:
```suggestion
  Switch, Xbox One and Xbox Series X/S (GDK) porting and publishing of Godot games (GDScript/C#).
```
I'm not sure there is a reason to use an ampersand here. Fine otherwise.

Reply:
Changed it just for brevity sake and not to have "and (...) and" in one line, but if you think it's better it can be like this.

Reply:
I think it's okay to have "and (...) and" here. Can add an Oxford comma to help with reading, or just replace it with a comma if you want.